README.rst

```diff
@@ -13,8 +13,8 @@ OpenWISP Notifications Module
 It is used to notify OpenWISP users about meaningful events in their network.
 It not only handles common tasks like selecting appropriate recipients for a notification,
 sending email notifications, etc. but also provide measures to customize those notifications with provision for
-configurable email templates, grouping of notifications to ease management and so on. 
-**openwisp-notifications** takes care of all this, so you can focus on what matters the most. 
+configurable email templates, grouping of notifications to ease management and so on.
+**openwisp-notifications** takes care of all this, so you can focus on what matters the most.
```

Comment:
Start to create a feature list like we have here https://github.com/openwisp/openwisp-ipam#available-features